plugins/woocommerce/package.json

```diff
@@ -529,8 +529,12 @@
 						".wp-env.json"
 					],
 					"events": [
-						"disabled"
-					]
+						"pull_request"
```

Comment:
@adimoldovan, is this supposed to run only against PRs? IIUC [unless we run against trunk](https://github.com/woocommerce/woocommerce/blob/27a85ecd51511a84988a49c3851b6b5071fb3314/.github/workflows/scripts/run-metrics.sh#L29-L31) the results won't be logged anywhere. It looks like the last log was on June 20: https://www.codevitals.run/project/woo

Reply:
My understanding is that we're ok with keeping it on push to trunk only (pdV5qK-zn-p2#comment-968). So the right event would be `push`.

Reply:
I remembered we discussed that right after submitting the comment, so I edited it. 😄 Didn't make it in time before you responded, though! Anyway, as mentioned above, let's switch to push before this is merged.